providers/plunk/package.json

```
@@ -0,0 +1,83 @@
+{
+  "name": "@novu/plunk",
+  "version": "0.16.0",
```

We are in version 0.16.3 right now: https://github.com/novuhq/novu/blob/next/apps/api/package.json
```suggestion
  "version": "0.16.3",
```
Not your fault as the script to generate the skeleton of the provider was set to 0.16.0.

---

That's okay. I've updated it